docs/user_guide/mitigation/postprocessing.inc

```diff
@@ -0,0 +1,6 @@
+.. _postprocessing:
+
+Postprocessing
+--------------
+
+.. currentmodule:: fairlearn.postprocessing
```

Comment:
I was shocked here that my user guide is gone until I realized that the PR hasn't been completed yet. We should  get to that as well.